{"uuid": "487fa62c-60e8-4213-ad3e-b16a3d819a1b", "children": ["ada81ac4-c4fb-420b-b242-9bb85ec94254"], "befores": [{"name": "create_period_of_disease", "status": "failed", "statusDetails": {"message": "AssertionError: assert 400 == 200\n +  where 400 = <Response [400]>.status_code\n +  and   200 = status.HTTP_200_OK\n", "trace": "  File \"/Users/ahmed/Documents/Zewail/grad/back/GreadProject-Backend-fastapi/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/Users/ahmed/Documents/Zewail/grad/back/GreadProject-Backend-fastapi/venv/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Users/ahmed/Documents/Zewail/grad/back/GreadProject-Backend-fastapi/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ahmed/Documents/Zewail/grad/back/GreadProject-Backend-fastapi/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/ahmed/Documents/Zewail/grad/back/GreadProject-Backend-fastapi/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 877, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/ahmed/Documents/Zewail/grad/back/GreadProject-Backend-fastapi/test/test_periodOfDisease.py\", line 52, in create_period_of_disease\n    assert response.status_code == status.HTTP_200_OK\n"}, "start": 1717595743324, "stop": 1717595743587}], "start": 1717595743324, "stop": 1717595743597}